[2021-07-16 16:10:55,018] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:09:57.638214+00:00 [queued]>
[2021-07-16 16:10:55,069] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:09:57.638214+00:00 [queued]>
[2021-07-16 16:10:55,070] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 16:10:55,071] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-07-16 16:10:55,072] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-07-16 16:10:55,098] {taskinstance.py:1087} INFO - Executing <Task(TriggerDagRunOperator): restart_test> on 2021-07-16T16:09:57.638214+00:00
[2021-07-16 16:10:55,121] {standard_task_runner.py:52} INFO - Started process 6704 to run task
[2021-07-16 16:10:55,132] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'psotgrressdftest', 'restart_test', '2021-07-16T16:09:57.638214+00:00', '--job-id', '957', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/POSTGRESRSERS.py', '--cfg-path', '/tmp/tmpsrsi02cj', '--error-file', '/tmp/tmp7ebirdr9']
[2021-07-16 16:10:55,134] {standard_task_runner.py:77} INFO - Job 957: Subtask restart_test
[2021-07-16 16:10:55,267] {logging_mixin.py:104} INFO - Running <TaskInstance: psotgrressdftest.restart_test 2021-07-16T16:09:57.638214+00:00 [running]> on host 13346afafa5b
[2021-07-16 16:10:55,404] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=psotgrressdftest
AIRFLOW_CTX_TASK_ID=restart_test
AIRFLOW_CTX_EXECUTION_DATE=2021-07-16T16:09:57.638214+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-16T16:09:55.085887+00:00
[2021-07-16 16:11:01,777] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=psotgrressdftest, task_id=restart_test, execution_date=20210716T160957, start_date=20210716T161055, end_date=20210716T161101
[2021-07-16 16:11:01,826] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-16 16:11:01,872] {local_task_job.py:151} INFO - Task exited with return code 0
